plasmapy/physics/tests/test_transport.py

```diff
@@ -42,56 +42,147 @@ def count_decimal_places(digits):
     return len(fractional)
 
 
-def test_Coulomb_logarithm():
-
-    n_e = np.array([1e9, 1e9, 1e24]) * u.cm**-3
-    T = np.array([1e2, 1e7, 1e8]) * u.K
-    Lambda = np.array([5.56, 21.45, 6.48])
-    particles = ('e', 'p')
-
-    for i in range(3):
-        assert np.isclose(Coulomb_logarithm(T[i], n_e[i], particles),
-                          Lambda[i], atol=0.01)
-
-    assert np.isclose(Coulomb_logarithm(1 * u.eV, 5 * u.m**-3, ('e', 'e')),
-                      Coulomb_logarithm(11604.5220 * u.K,
-                                        5 * u.m**-3,
-                                        ('e', 'e')))
-
-    assert np.isclose(Coulomb_logarithm(1e2 * u.K, 1e9 * u.cm**-3, ('e', 'p')),
-                      5.56, atol=0.01)
-
-    assert np.isclose(Coulomb_logarithm(1e7 * u.K, 1e9 * u.cm**-3, ('e', 'p')),
-                      21.45, atol=0.1)
-
-    assert np.isclose(Coulomb_logarithm(1e8 * u.K,
-                                        1e24 * u.cm**-3,
-                                        ('e', 'p')),
-                      6.48, atol=0.01)
-
-    assert np.allclose(Coulomb_logarithm(T, n_e, particles), Lambda, atol=0.01)
-
-    assert np.isclose(Coulomb_logarithm(1e5 * u.K, 5 * u.m**-3, ('e', 'e'),
-                                        V=1e4 * u.m / u.s), 21.379082011)
-
-    with pytest.warns(RelativityWarning):
-        Coulomb_logarithm(1e5 * u.K, 1 * u.m**-3, ('e', 'p'), 0.9 * c)
-
-    with pytest.raises(RelativityError):
-        Coulomb_logarithm(1e5 * u.K, 1 * u.m**-3, ('e', 'p'), 1.0 * c)
-
-    with pytest.raises(u.UnitConversionError):
-        Coulomb_logarithm(1e5 * u.g, 1 * u.m**-3,
+class Test_Coulomb_logarithm(object):
```

Comment:
So... we're simply dumping the old tests here? That seems kind of weird to me.

Reply:
So I moved everything into a class with each test having its own method. That way if one test fails it will still run the other tests. I've kept the old warning and error tests  as they were useful. I didn't find the value checking tests very useful, so I replaced them with known values from Chen for a wide range of plasma parameters.